charts/opentelemetry-ebpf/values.yaml

```
@@ -0,0 +1,275 @@
+# Default values for opentelemetry-ebpf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+clusterName: ""
+
+images:
+  tag: "v0.10.0"
+  repository: otel
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+
+resources: {}
+
+otelCollector:
+  address: ""
+  port: ""
+
+log:
+  console: false
+  # possible values: { error | warning | info | debug | trace }
+  level: warning
+
+debug:
+  enabled: false
+  storeMinidump: false
+  sendUnplannedExitMetric: false
+
+kernelCollector:
+  enabled: true
+  serviceAccount:
+    create: true
+    name: ""
+  image:
+    repository: ""
+    tag: ""
+    name: opentelemetry-ebpf-kernel-collector
```

If the above is changed to `.Values.image.registry`, then the `name` here is called `reposotory`.

```suggestion
  image:
    repository:  opentelemetry-ebpf-kernel-collector
    # Overrides the image tag whose default is the chart appVersion.
    tag: ""
```